content/blog/2020/05/2020-05-07-docker-windows-agents.adoc

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Windows Docker Agent Images"
+tags:
+- announcement
+- docker
+- platform-sig
+author: slide_o_mix
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We would like to announce the availability of Windows agent images for Docker.
+
+image:/images/docker/dockerJenkins.png[Jenkins and Docker, role=center, float=right, height=224]
+
+== New images
+
+* link:https://hub.docker.com/r/jenkins/agent[jenkins/agent] is a basic agent which bundles the agent.jar for agent <= => master communication. This is most useful as a base image for other images.
```

Comment:
```suggestion
* link:https://hub.docker.com/r/jenkins/agent[jenkins/agent] is a basic agent which bundles the agent.jar for agent <= => master communication. This is most useful as a base image for other images.
  Windows images are available starting from version link:https://github.com/jenkinsci/docker-agent/releases/tag/4.3-4[4.3-4]
```

Reply:
Not sure about Windows usage, but at least the Linux flavor is usable as is via the command launcher.